tests/tests/control_atm_aerosols

```diff
@@ -0,0 +1,83 @@
+###############################################################################
+#
+#  global control test with prognostic aerosols: GFSv16 FV3-GOCART at C96L127
+#
+###############################################################################
+
+export TEST_DESCR="Compare global results for prognostic aerosols with previous trunk version"
+
+export CNTL_DIR=control_atm_aerosols
+
+export LIST_FILES="sfcf000.nc \
+                   sfcf024.nc \
+                   atmf000.nc \
+                   atmf024.nc \
+                   GFSFLX.GrbF00 \
+                   GFSFLX.GrbF24 \
+                   GFSPRS.GrbF00 \
+                   GFSPRS.GrbF24 \
+                   gocart.inst_aod.20210322_0700z.nc4 \
+                   gocart.inst_aod.20210323_0600z.nc4 \
+                   gocart.tavg_cmass.20210322_0730z.nc4 \
+                   gocart.tavg_cmass.20210323_0430z.nc4 \
+                   RESTART/coupler.res \
+                   RESTART/fv_core.res.nc \
+                   RESTART/fv_core.res.tile1.nc \
+                   RESTART/fv_core.res.tile2.nc \
+                   RESTART/fv_core.res.tile3.nc \
+                   RESTART/fv_core.res.tile4.nc \
+                   RESTART/fv_core.res.tile5.nc \
+                   RESTART/fv_core.res.tile6.nc \
+                   RESTART/fv_srf_wnd.res.tile1.nc \
+                   RESTART/fv_srf_wnd.res.tile2.nc \
+                   RESTART/fv_srf_wnd.res.tile3.nc \
+                   RESTART/fv_srf_wnd.res.tile4.nc \
+                   RESTART/fv_srf_wnd.res.tile5.nc \
+                   RESTART/fv_srf_wnd.res.tile6.nc \
+                   RESTART/fv_tracer.res.tile1.nc \
+                   RESTART/fv_tracer.res.tile2.nc \
+                   RESTART/fv_tracer.res.tile3.nc \
+                   RESTART/fv_tracer.res.tile4.nc \
+                   RESTART/fv_tracer.res.tile5.nc \
+                   RESTART/fv_tracer.res.tile6.nc \
+                   RESTART/phy_data.tile1.nc \
+                   RESTART/phy_data.tile2.nc \
+                   RESTART/phy_data.tile3.nc \
+                   RESTART/phy_data.tile4.nc \
+                   RESTART/phy_data.tile5.nc \
+                   RESTART/phy_data.tile6.nc \
+                   RESTART/sfc_data.tile1.nc \
+                   RESTART/sfc_data.tile2.nc \
+                   RESTART/sfc_data.tile3.nc \
+                   RESTART/sfc_data.tile4.nc \
+                   RESTART/sfc_data.tile5.nc \
+                   RESTART/sfc_data.tile6.nc"
+
+export_fv3
+export NPZ=127
+export NPZP=128
+export DT_ATMOS=720
+export SYEAR=2021
+export SMONTH=03
+export SDAY=22
+export SHOUR=06
+export RESTART_INTERVAL=12
+export OUTPUT_GRID='gaussian_grid'
+export NSTF_NAME='2,0,0,0,0'
+export WRITE_DOPOST=.true.
+export IAER=5111
+export FSCAV_AERO='"*:0.6", "seas1:1.0", "seas2:1.0", "seas3:1.0", "seas4:1.0", "seas5:1.0"'
```

Comment:
@rmontuoro I thought the test is going to use RAS scavenging from GOCART, I do not see RAS is specified, so we are using SAS scavenging for aerosol particles?

Reply:
@junwang-noaa - The target physics suite for prognostic aerosols is `FV3_GFS_v16`, which includes SAS. This suite is used in the included regression test. I was not aware that RAS will be used with prognostic aerosols. However, we can update to RAS later on.